lib/runner.js

```diff
@@ -96,7 +96,9 @@ Runner.prototype.run = function () {
 		testCount: 0
 	};
 
-	return Promise.resolve(this.tests.buildPhases(this._addTestResult, this._bail).run()).then(function () {
+	this.tests.on('test', this._addTestResult);
+
+	return Promise.resolve(this.tests.build(this._bail).run()).then(function () {
```

Comment:
I doubt that'll be noticeable given the cost of spawning a new process in order to run the tests.